prediction_market_agent/agents/microchain_agent/prompts.py

```diff
@@ -7,7 +7,8 @@
 
 {{engine_help}}
 
-Only output valid Python function calls, without code formatting characters.
+Only output valid Python function calls, without code formatting characters, without any other text.
+Only output a single function call per message.
```

Comment:
This was interesting, it wanted to call multiple function in a single run. Could be interesting if we would support it.

@coderabbitai Please create an issue to try this out.